src/utils.ts

```diff
@@ -9,3 +9,25 @@ export function isObject(obj: unknown): obj is Record<string, unknown> {
 export function getMemberVarName(objectVarName: string, key: string): string {
   return objectVarName !== '' ? `${objectVarName}[${JSON.stringify(key)}]` : key;
 }
+
+export function intersection<T>(setA: Set<T>, setB: Set<T>): Set<T> {
+  // From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#implementing_basic_set_operations
+
+  const _intersection = new Set<T>();
```

Comment:
Though the example code use `_` as a prefix, I think it is not common. So please remove `_` from the variable name.